CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+# PrestaShop Code of Conduct
+
+English version 
+
+## Purpose 
+
+This Code of Conduct outlines how to behave to participate in the PrestaShop open source project (the “Project”) as well as prohibited behaviours and reporting procedure for damaging behaviours and/or situations. 
+
+The Code of Conduct aims to create a positive, inspiring and welcoming community, whether you are a beginner or an expert, asking questions or seeking to share your knowledge. 
+
+
+## Scope of application
+
+This Code of Conduct applies to all people contributing to the Project, especially through online contributions, whether it is repositories, forum posts or on social media. 
+
+It also applies to public spaces, during PrestaShop events, aka the PSD, PS Connect, meetups or any other event during which an individual or a group of individuals represents the Project. 
+
+The Code of Conduct applies to any member of the PrestaShop community, whether it is PrestaShop employees, partners, ambassadors, moderators or any other members on the contributory sites and PrestaShop social media. 
+
+
+## Expected behaviours 
+
+The Code of conduct aiming to create a positive and benevolent community, the following principles must be followed in order to have constructive and respectful exchanges : 
```

Comment:
This sentence is a bit weird no? The structure doesn't feel English.

Proposition: 
_The Code of conduct ~aiming~ **aims** to create a positive and benevolent community, **thus** the following principles must be followed in order to have constructive and respectful exchanges_

or just 

_The following principles must be followed in order to have constructive and respectful exchanges_

Reply:
I will update that. thanks.